pytorch_lightning/core/saving.py

```diff
@@ -48,6 +48,37 @@ def on_hpc_load(self, checkpoint: Dict[str, Any]) -> None:
         """
 
 
+def update_hparams(hparams, updates):
+    """
+    Overrides hparams with new values
+
+    >>> hparams = {'a', {'b': 2}, 'c': 1}
+    >>> updates = {'a': {'b': 4}, 'c': 7}
+    >>> update_hparams(hparams, updates)
+    >>> assert hparams['a']['b'] == 4, 'update hparams failed'
+    >>> assert hparams['c'] == 7, 'update hparams failed'
```

Comment:
```suggestion
    >>> hparams = {'a', {'b': 2}, 'c': 1}
    >>> updates = {'a': {'b': 4}, 'c': 7}
    >>> update_hparams(hparams, updates)
    >>> hparams['a']['b']
    4
    >>> hparams['c']
    7
```
This is they way doctests are expected to test things (test + document usage at the same time) @williamFalcon

Reply:
this is the way as it serves also as meaningful example... :]